TAGradingServer/account/print/print_checkpoints_gradeable.php

```diff
@@ -40,8 +41,24 @@
     </tr>
 HTML;
 
-$query = ($grade_by_reg_section) ? "SELECT * FROM users WHERE registration_section=? AND user_group=? ORDER BY user_id"
-                                 : "SELECT * FROM users WHERE rotating_section=? AND user_group=? ORDER BY user_id";
+$query = "SELECT * FROM users WHERE ";
+if($grade_by_reg_section){
+    $query .= "registration_section=?";
+}
+else{
+    $query .= "rotating_section=?";
+}
+$query .= " AND user_group=? ORDER BY ";
+if($sort_by === 'first'){
+    $query .= "user_firstname";
+}
+elseif($sort_by === 'last'){
+    $query .= "user_lastname";
+}
+else{
+    $query .= "user_id";
+}
+
```

Comment:
We don't want to sort just on firstname or just one lastname or just on id as this can lead to unpredictable behavior if we have students who have matching first names and last names (it'll be sorted by insertion order).

I'd like to see this be:
```
$sort = array();
if ($sort_by === 'first') {
    $sort[] = 'user_firstname';
    $sort[] = 'user_lastname';
}
else if ($sort_by === 'last') {
    $sort[] = 'user_lastname';
}
$sort[] = 'user_id';
$query .= implode(', ', $sort);
```